packages/workspace-store/src/schemas/extensions/example/x-is-disabled.ts

```diff
@@ -0,0 +1,13 @@
+import { Type } from '@scalar/typebox'
+
+/**
+ * An OpenAPI extension to mark examples as enabled or disabled
```

Comment:
Can you add to this comment so I understand why an example should be disabled or what the consequence would be?

Just saying disabled disables it, doesn’t help present me, and future me even less. 👀 

Same for all those comments in the added extensions.